fjord/feedback/static/js/android_feedback.js

```diff
@@ -0,0 +1,66 @@
+(function($) {
+  'use strict';
+
+  // settings
+  var browserReviewUrl = 'https://play.google.com/store/apps/details?id=org.mozilla.CHANNEL#details-reviews';
+  var deviceReviewUrl = 'market://details?id=org.mozilla.CHANNEL';
+
+  // initialize
+  var reviewUrl, lastUrl = '';
+  var url = decodeURIComponent(window.location.href);
+
+
+  /*
+    Decide whether they're on Fx Android or not
+    (Fx Android's feedback prompt includes a utm param)
+  */
+  var fromDevice = /source=feedback-prompt/.test(url);
+  if (fromDevice) {
+    reviewUrl = deviceReviewUrl;
+    /* 
+      TODO: When bug 1237388 is resolved, remove #play-store here.
+      If url params are as assumed below, it'll Just Work(tm)!
+    */
+    $('#maybe-later, #no-thanks, #play-store').click(
```

Comment:
Using `$().on` is preferred over `click` since it handles the event as it bubbles and can deal with JS changing the page:

```js
$(document).on('click', '#maybe-later, #no-thanks, #play-store', function(e) {

});
```